svg/svg.go

```diff
@@ -92,7 +94,45 @@ func NewWindowControls(r float64, x, y float64) *etree.Element {
 		circle.CreateAttr("fill", color)
 		bar.AddChild(circle)
 	}
-	return bar
+	controlsWidth := float64(len(bar.ChildElements()))*float64(x) + float64(r)*2
+	return bar, controlsWidth
+}
+
+// NewWindowTitle returns a title element with the given text.
+func NewWindowTitle(positions []float64, position string, fs float64, ff, text string, s *chroma.Style) (*etree.Element, error) {
```

Comment:
Or maybe a config object to simplify everything

Reply:
@ccoVeille I was trying to do that but when using `Config` inside `./svg/svg.go` it says undefined and then I tried importing it from `github.com/charmbracelet/freeze` but says `import cycle not allowed`:

```bash
svg/svg.go:11:2: import "github.com/charmbracelet/freeze" is a program, not an importable package
package github.com/charmbracelet/freeze
        imports github.com/charmbracelet/freeze/svg
        imports github.com/charmbracelet/freeze: import cycle not allowed
```

I am missing something here?

Reply:
Do you really need an svg package? If not put everything in same folder/package it would fix your issue

Reply:
I don't really need it perse, but it's where `NewWindowControls` function is so I thought it would be a better fit to create it there. But I can make it so its within main